pkgs/development/libraries/thrift/0.10.nix

```diff
@@ -0,0 +1,39 @@
+{ stdenv, fetchurl, boost, zlib, libevent, openssl, python, pkgconfig, bison
+, flex, twisted
+}:
+
+stdenv.mkDerivation rec {
+  name = "thrift-${version}";
```

Comment:
Please replace `name` with `pname = "thrift";` as `name` is already defined elsewhere.